### DISCUSSION TITLE
Next.js tutorial step 1 (installing) currently not working due to missing dependency

### DISCUSSION THREAD

**Discussion by hacktisch** [vercel/next.js#15936] [Help] [answered]
EDIT: Issue not because of Nextjs, it is a temporary NPM issue as amedvedevajt said
------------------------------------------------

See the basic tutorial: https://nextjs.org/learn/basics/create-nextjs-app/setup

to install the starter, one must run:
`npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"`

This results in the following message:
```
npm ERR! 404  '@ampproject/toolbox-optimizer@2.2.0' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'next'
```

Not sure why this package isn't found in the NPM registry, I do find it on the npmjs.com site. So I'm not sure whether this is a problem on NPM's side, or the specific package in question.

**Comment by amedvedevajt** [comment, marked as answer]
It looks like npm issue. I have the same problem with deploy my project today.
https://status.npmjs.org/
